keyboards/satan/keymaps/vandheer/keymap.c

```diff
@@ -0,0 +1,210 @@
+
+/* Keymap _FN: Function Layer
+   * ,-----------------------------------------------------------.
+   * |   | F1|F2 |F3 |F4 |F5 |F6 |F7 |F8 |F9 |F10|F11|F12|  DEL  |
+   * |-----------------------------------------------------------|
+   * |     |   |   |   |   |   |   |   |   |   |   |   |   |     |
+   * |-----------------------------------------------------------|
+   * |Ctrl   |   |   |   |   |   |   |   |   |   |Vol-|Vol+|     |
+   * |-----------------------------------------------------------|
+   * |Shift   |   |   |   |   |   |   |   |   |   |   |          |
+   * |-----------------------------------------------------------|
+   * |_NA |Alt |Gui |                       |Cap |Alt |_FN |Macro|
+   * `-----------------------------------------------------------'
+   */
+[_FN] = LAYOUT_60_ansi(
+  LALT(LGUI(KC_ESC)),   KC_F1,    KC_F2,  KC_F3,  KC_F4,  KC_F5,  KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12, KC_DEL, \
+  __n__,    __n__,    __n__,  __n__,  __n__,  __n__,  __n__,    __n__,    __n__,    __n__,   __n__,    __n__,    __n__,  __n__,  \
+  _______,  __n__,    __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__, KC_VOLD,  KC_VOLU,          _______,\
+  _______,  __n__,    __n__,  __n__,  __n__,  __n__,  __n__,  __n__,    __n__,    __n__,    __n__,                      _______,\
+  _______,  _______,  _______,                     _______,                         _______,  _______,_______,            _______),
+
+  /* Keymap _MA: Macro Layer
+   * ,-----------------------------------------------------------.
+   * |   |   |   |   |   |   |   |   |   |   |   |   |   |       |
+   * |-----------------------------------------------------------|
+   * |     |   |   |   |   |   |   |   |   |   |   |   |   |     |
+   * |-----------------------------------------------------------|
+   * |       |   |   |   |   |   |   |   |   |   |   |   |       |
+   * |-----------------------------------------------------------|
+   * |        |   |   |   |   |   |   |   |   |   |   |          |
+   * |-----------------------------------------------------------|
+   * |    |    |    |                        |    |    |   |Macro|
+   * `-----------------------------------------------------------'
+   */
+[_MA] = LAYOUT_60_ansi(
+  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__, \
+  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__, \
+  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,          __n__,  \
+  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,  __n__,                  __n__, \
+  __n__,  __n__,  __n__,                           __n__,                         __n__,  __n__,  __n__,  _______),
+};
+
+enum function_id {
+    GRAVE_ON_ESC,
+    GRAVE_ON_TAB,
+};
+
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
Could you remove the `fn_actions` block?  All of it's features have been added as keycodes, and been deprecated in favor of `process_record_user`.